gradio/components/chatbot.py

```diff
@@ -185,10 +187,17 @@ def _preprocess_chat_messages(
         elif isinstance(chat_message, str):
             return chat_message
         elif isinstance(chat_message, ComponentMessage):
-            component = import_component_and_data(chat_message.component.capitalize())
+            capitalized_component = (
+                chat_message.component.upper()
+                if chat_message.component in ("json", "html")
+                else chat_message.component.capitalize()
+            )
+            component = import_component_and_data(capitalized_component)
```

Comment:
To handle. `gr.HTML` and `gr.JSON` which don't follow the same pattern as others.

Reply:
We should probably think of a better way to handle this at some point.